Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +162,37 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    _renderTimestamp()
+    {
+        if (!this._timestamp) {
+            if (this._timestampElement) {
+                this._timestampElement.remove();
+                this._timestampElement = null;
+            }
+            return;
+        }
+
+        if (!this._timestampElement) {
+            this._timestampElement = document.createElement("span");
```

Comment:
The constructor should set `this._timestampElement = null` to prevent us from "reshaping" this class at runtime. It also helps us keep track of what the various member variables are in a class if the constructor explicitly sets them (even if it sets them to null).